staticaddr/protocol_version.go

```diff
@@ -0,0 +1,63 @@
+package staticaddr
+
+import (
+	"math"
+
+	looprpc "github.com/lightninglabs/loop/swapserverrpc"
+)
+
+// AddressProtocolVersion represents the protocol version (declared on rpc
+// level) that the client declared to us.
+type AddressProtocolVersion uint32
+
+const (
+	// ProtocolVersionLegacy indicates that the client is a legacy version
+	// that did not report its protocol version.
+	ProtocolVersionLegacy AddressProtocolVersion = 0
```

Comment:
This (and unrecorded) can't happen as any client we release that supports static address will report their static addr protocol version. So I'd suggest to just add the first version with an appropriate name, such as STATIC_ADDR_V1 or anything you feel is better (value can be 0 or 1 too).